elastic/apm-server
# Collect apm-server memstats correctly from expvar

## Motivation/summary

Benchmark result is currently not collecting apm-server data for non-extra metrics.

## Checklist

~~- [ ] Update [CHANGELOG.asciidoc](https://github.com/elastic/apm-server/blob/main/CHANGELOG.asciidoc)~~
~~- [ ] Update [package changelog.yml](https://github.com/elastic/apm-server/blob/main/apmpackage/apm/changelog.yml) (only if changes to `apmpackage` have been made)~~
~~- [ ] Documentation has been updated~~

## How to test these changes

1. Run `apmbench` test as per the doc [here](https://github.com/elastic/apm-server/blob/main/TESTING.md)
2. Observe the benchmark result and compare them with expvar metrics of the target apm-server


## Related issues
